nextflow.config

```diff
@@ -24,9 +24,9 @@ params {
     gtf               = null
 
     // alevin-fry parameters (simpleaf)
-    simpleaf_rlen     = 91
+    simpleaf_index      = null
     barcode_whitelist = null
-    salmon_index      = null
+    simpleaf_umi_resolution = "cr-like"
```

Comment:
What does this parameter do? Is this a reasonable default, or should it be set based on the protocol used? We have a protocols.json file somewhere that already sets other parameters based on the protocol.

Reply:
It's all good then.